testsuite/test_cim_operations2.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
```

Comment:
I don't think we should move this into a new test file. See my explanation in the conversation of this PR.